cluster/src/main/java/org/apache/iotdb/cluster/client/ClientCategory.java

```diff
@@ -0,0 +1,19 @@
+package org.apache.iotdb.cluster.client;
+
+public enum ClientCategory {
+  META("MetaClient"),
+  META_HEARTBEAT("MetaHeartbeatClient"),
+  DATA("DataClient"),
+  DATA_HEARTBEAT("DataHeartbeatClient"),
+  SINGLE_MASTER("SingleMasterClient");
```

Comment:
what is this for?

Reply:
`SINGLE_MASTER` is a special async data client type in which all connection share a single selector and is only. As I don't intend to break original logic too much, so this one is kept.